package.json

```diff
@@ -35,6 +35,7 @@
     "@fontsource/roboto": "^5.0.8",
     "@mui/icons-material": "^5.15.2",
     "@mui/material": "^5.15.2",
+    "@mui/system": "^5.15.5",
```

Comment:
is this needed? the `SxProps` can also be imported from `@mui/material`

Reply:
I see, I can adjust the import